src/js/NasaTable/NasaTable.js

```diff
@@ -26,6 +28,7 @@ class NasaTable extends Component {
       clickedImageName: '',
       isDescendingSort: true,
       sortKey: 'imgName',
+      notification: false,
```

Comment:
Rename it to look like a boolean value please. E.g. isNotificationShown or shouldShowNotification